docs/api-reference/org/create-new-project.mdx

```diff
@@ -0,0 +1,3 @@
+---
+openapi: post /api/v1/org/projects/create
+---
```

Comment:
Consider adding rate limiting information and example request/response payloads in the API documentation to make it more user-friendly.